tests/phpunit/Unit/Annotator/ExtraPropertyAnnotatorTest.php

```diff
@@ -371,4 +371,38 @@ private function newExtraPropertyAnnotatorInstanceFor( $externalId ) {
 		return $instance;
 	}
 
+	public function testPropertyAnnotation_PAGELGTH() {
```

Comment:
Thanks a lot. Things like this are just out of my reach. I was already happy that I thought of adding this special property to the registration test. Programmers really have my respect.